### PR TITLE
[DebuggerV2] Highlight inputs of graph-execution event

### PR DESCRIPTION
**Pull request by caisq**
* Motivation for features / changes
  * Improve the graph debugging of DebuggerV2
  * Highlight immediate input(s) of the graph tensor being focused on
     (through clicking in the graph execution panel)
* Technical description of changes
  * Add a selector (`getFocusedGraphExecutionInputIndices()`) that makes
     a best-effort attempt to find the indices of the graph-execution events that
     are the immediate inputs of the graph-execution event currently being
     focused on.
  * In the `GraphExecutionComponent`, highlight these input events with a
     light-yellow background color (fff099). 
  * In the `GraphComponent`, highlight the input tensors with the same
    background color. 
  * When users click tensor names in the `GraphExecutionComponent`,
    let the clicked on be the focused on, and scroll it to the center smoothly.
* Screenshots of UI changes
  * image
* Detailed steps to verify changes work correctly (as executed by you)
  * Unit tests added for new actions, selectors, and reducers.
  * Testing against a real tfdbg2 dump dir (see screenshot).

### CODE REVIEW

**Comment by stephanwlee** [comment]
Also, please mark the PR as ready for review (currently, it is a draft)

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors_test.ts [hunk 1]
```diff
@@ -1240,6 +1240,307 @@ describe('debugger selectors', () => {
...
+              ]),
+            },
+            focusedOp: {
+              graphId: 'g1',
+              opName: 'op3',
+            },
+          }),
+        })
+      );
+      expect(getFocusedGraphExecutionInputIndices(state)).toEqual([10, 6]);
```
[the remark this answers is not in the document]
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors_test.ts:1451]: This is good feedback. The instances of `op[n]Info` are actually not as critical to the tests as the `graphExecutionData` in these tests. They are also repetitive across the `it`s. So I've refactored them into a single `const`. This avoids distracting code readers. As for the `graphExecutionData`, their contents have in-line comments to facilitate understanding already. I did another pass over them to make sure they are consistent and easy to follow.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts [hunk 2]
```diff
@@ -52,6 +51,7 @@ import {
 // https://github.com/bazelbuild/rules_nodejs/issues/1013
 /** @typehack */ import * as _typeHackSelector from '@ngrx/store/src/selector';
 /** @typehack */ import * as _typeHackStore from '@ngrx/store/store';
+import {connectableObservableDescriptor} from 'rxjs/internal/observable/ConnectableObservable';
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts:54]: Unused. If you want to keep this, please put the typehack preamble.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_component.css [hunk 3]
```diff
@@ -55,10 +50,12 @@ limitations under the License.
 }
 
 .input-slot-header {
+  /** NOTE: If this is changed, change the corresponding color in
+      graph_execution_component.css as well. */
+  background-color: #fff099;
```
[the remark this answers is not in the document]
Comment (stephanwlee) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph/graph_component.css:56]: ACK.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts [hunk 4]
```diff
@@ -174,7 +174,7 @@ describe('Graph Executions Container', () => {
     })
   );
 
-  it('clicking the tensor name buttons dispatches graphOpFocused', fakeAsync(() => {
+  it('clicking tensor name dispatches graphExecutionFocused', fakeAsync(() => {
```
Comment (stephanwlee) [inline tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts:177]: `dispatches graphExecutionFocused when tensor name is clicked`.

Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/graph_executions/graph_executions_container_test.ts:177]: Done.

File: tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts (hunk 2 above)
Comment (caisq) [reply tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/store/debugger_selectors.ts:54]: Ah - this is one of those annoying auto-imports due to VSCode + my typo. Removed. Thanks for catching it.

**Pull request merged**